### ISSUE TITLE
Could you provide the pre-trained models or training codes in Sec. 5.4-5.6 ?

### ISSUE THREAD

**Issue by YuheD** [OpenGVLab/Multitask-Model-Selector#3]
Hello  @FanqingM , Thanks for your interesting work.

When I reproduced your work, I noticed that your pre-trained model and fine-tuning accuracy were obtained by yourself, would you be willing to provide the pre-trained model (or the training code of the model) and the code for model fine-tuning?

Best regards.
